lib/pacemaker/pcmk_sched_bundle.c

```diff
@@ -482,13 +480,14 @@ pcmk__bundle_rsc_colocation_rh(pe_resource_t *dependent, pe_resource_t *primary,
     } else if(constraint->dependent->variant > pe_group) {
         pe_resource_t *primary_replica = compatible_replica(dependent, primary,
                                                             RSC_ROLE_UNKNOWN,
-                                                            FALSE, data_set);
+                                                            FALSE,
+                                                            dependent->cluster);
```

Comment:
Nice. I wonder if this is doable later in `pcmk__native_allocate()` via `rsc->cluster`, or if `data_set` points to a separate object there and needs to stay that way.

Reply:
There's never more than one data_set. Actually compatible_replica() doesn't need the argument at all anymore, but I'll take care of that when I get to that file.